packages/constants/src/cycle.ts

```diff
@@ -24,33 +8,33 @@ export const CYCLE_STATUS: {
   bgColor: string;
 }[] = [
   {
-    label: "day left",
+    label: "project_cycles.status.days_left",
```

Comment:
We will have to change it to `labelTranslationKey` and `titleTranslationKey` respectively.